### PR TITLE
Rename packages name

### LINKED ISSUE #166: Rename packages from io.xstefank -> org.wildfly

### PR DESCRIPTION
**Pull request by Vladimir284**
Resolves #166

### CODE REVIEW

**Review by The-Huginn — APPROVED** [review]
LGTM, thanks @Vladimir284 for the nice work :star_struck:

File: src/main/java/org/wildfly/bot/util/PullRequestMergableProcessor.java [hunk 1]
```diff
@@ -20,26 +21,21 @@
...
-
 /**
  * This class was designed to retrieve mergable status for all pull
  * requests in a repository upon {@code io.quarkiverse.githubapp.event.Push} event.
  * This results in querying and listing all repository's open pull requests.
  * After a certain timeout it will re-query the repository's pull requests
  * again and update these pull requests if GitHub has updated the mergable
  * status for individual pull requests. To adjust the timeout time please
- * see {@see io.xstefank.wildfly.bot.config.WildFlyBotConfig#timeout()}
+ * see {@see config.org.wildfly.bot.WildFlyBotConfig#timeout()}
```
Comment (xstefank) [inline src/main/java/org/wildfly/bot/util/PullRequestMergableProcessor.java:31]:
```suggestion
 * see {@see org.wildfly.bot.config.WildFlyBotConfig#timeout()}
```

File: src/main/java/org/wildfly/bot/util/PullRequestMergableProcessor.java [hunk 2]
```diff
@@ -20,26 +21,21 @@
...
  * <p>
  * In case a re-queried pull request fails, it will be only logged.
  * <p>
  * Note: Do not call githubProcessor.LOG.setPullRequest inside parallel
  * Uni-s, i.e. inside the parameter `uniToExecute` in method
  * {@code combineUnis(Function<GHPullRequest, Uni<?>> uniToExecute)}
- * due to possible race condition of the {@code io.xstefank.wildfly.bot.util.PullRequestLogger}.
+ * due to possible race condition of the {@code util.org.wildfly.bot.PullRequestLogger}.
```
Comment (xstefank) [inline src/main/java/org/wildfly/bot/util/PullRequestMergableProcessor.java:38]:
```suggestion
 * due to possible race condition of the {@code org.wildfly.bot.util.PullRequestLogger}.
```

**Review by xstefank — APPROVED** [review]
Just a small typo in JavaDoc so I'm willing to merge this and fix this later.

**Comment by xstefank** [comment]
Thanks @Vladimir284!

**Pull request merged**